app/views/govuk_publishing_components/components/_contents_list.html.erb

```diff
@@ -44,8 +43,12 @@
       <% contents.each.with_index(1) do |contents_item, position| %>
         <li class="<%= cl_helper.list_item_classes(contents_item, false) %>" <%= "aria-current=true" if contents_item[:active] %>>
           <% link_text = format_numbers ? cl_helper.wrap_numbers_with_spans(contents_item[:text]) : contents_item[:text]
-             ga4_link_data = nil
-             ga4_link_data = { "index": { "index_link": position }, "index_total": contents.length }.to_json if ga4_tracking
+             if ga4_data
```

Comment:
What do you think about putting `if ga4_tracking` here instead of `if ga4_data`? And then I think we could remove the declaration `ga4_data = nil` (currently on line 16)